packages/block-library/src/query/utils.js

```diff
@@ -10,6 +10,13 @@ import { useSelect } from '@wordpress/data';
 import { useMemo } from '@wordpress/element';
 import { store as coreStore } from '@wordpress/core-data';
 import { decodeEntities } from '@wordpress/html-entities';
+import { cloneBlock } from '@wordpress/blocks';
+
+/**
+ * @typedef TransformedBlocksFromPattern
```

Comment:
Nit: why make this a typedef? It's referenced once. Personally, I'd prefer to read a slightly messier but immediate `@return {{ newBlocks: WPBlock[], queryClientIds: string[] }}` than looking up and grasping a one-off type.

Reply:
Done, thanks!